src/testMod/java/io/github/cottonmc/test/client/ScrollBarTestGui.java

```diff
@@ -0,0 +1,33 @@
+package io.github.cottonmc.test.client;
+
+import io.github.cottonmc.cotton.gui.client.LightweightGuiDescription;
+import io.github.cottonmc.cotton.gui.widget.WPlainPanel;
+import io.github.cottonmc.cotton.gui.widget.WScrollBar;
+import io.github.cottonmc.cotton.gui.widget.data.Axis;
+import io.github.cottonmc.cotton.gui.widget.data.Insets;
+
+import net.fabricmc.fabric.api.util.TriState;
+
+public class ScrollBarTestGui extends LightweightGuiDescription {
+	private boolean darkMode = false;
```

Comment:
This field is never written to, so I don't really see a point of having it. You could add a toggle button that changes its value, or just rely on the global light/dark mode setting.